### PR TITLE
Fix drop follower behaviour.

### PR DESCRIPTION
**Pull request by neunhoef**
This is a backport of a bugfix for devel.

### CODE REVIEW

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10139/

File: arangod/Utils/CollectionNameResolver.cpp [hunk 1]
```diff
@@ -223,34 +223,19 @@ std::string CollectionNameResolver::getCollectionNameCluster(TRI_voc_cid_t cid)
...
-    } else {
-      // most likely collection not found. now try again
-      _vocbase.server().getFeature<ClusterFeature>().clusterInfo().flush();
     }
   }
 
-  LOG_TOPIC("817e8", DEBUG, arangodb::Logger::FIXME)
-      << "CollectionNameResolver: was not able to resolve id " << cid;
+  LOG_TOPIC_IF("817e8", DEBUG, arangodb::Logger::FIXME, name == ::UNKNOWN)
+  << "CollectionNameResolver: was not able to resolve id " << cid;
```
Comment (maierlars) [inline arangod/Utils/CollectionNameResolver.cpp:236]:
```suggestion
      << "CollectionNameResolver: was not able to resolve id " << cid;
```
:scream:

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10156/

**Comment by KVS85** [comment]
Tests blue.

**Pull request merged**